index.html

```diff
@@ -3829,18 +3493,396 @@ <h2>
+        particular member and what to do when a type does not match what is
+        expected.
       </p>
+
+      <ul class="index" data-sort="">
+        <li>
+          [[!URL]] defines the following terms:
+          <ul>
+            <li>
+              <dfn><a data-cite="!URL#concept-url">URL</a></dfn>
+            </li>
+            <li>
+              <dfn data-lt="parse|parsing|URL parsing"><a data-cite=
+              "!URL#concept-url-parser">URL parser</a></dfn>
+            </li>
+          </ul>
+        </li>
+
+        <li>
+          [[!SCREEN-ORIENTATION]] defines the following terms:
+          <ul>
+            <li>
+              <a data-cite="!SCREEN-ORIENTATION#dfn-default-orientation"><dfn>
+              default orientation</dfn></a>
+            </li>
+            <li>
+              <a data-cite=
+              "!SCREEN-ORIENTATION#idl-def-OrientationLockType">
+              <dfn>OrientationLockType</dfn></a>
+            </li>
+          </ul>
+        </li>
+
+        <li>
+          [[!CSS-SYNTAX-3]] defines the following terms:
+          <ul>
+            <li>
+              <a data-cite="CSS-SYNTAX-3#parse-component-value">
+              <dfn>parse a component value</dfn></a>
+            </li>
+          </ul>
+        </li>
+
+        <li>
+          [[!CSP3]] defines the following terms:
+          <ul>
+            <li>
+              <a data-cite="!CSP3#directive-manifest-src"><dfn>manifest-src</dfn> directive</a>
+            </li>
+            <li>
+              <a data-cite="!CSP3#directive-img-src"><dfn>img-src</dfn> directive</a>
+            </li>
+            <li>
+              <a data-cite="!CSP3#directive-default-src"><dfn>default-src</dfn> directive</a>
+            </li>
+          </ul>
+        </li>
+
+        <li>
+          [[!SECURE-CONTEXTS]] defines the following terms:
+          <ul>
+            <li>
+              <a href=
+              "https://w3c.github.io/webappsec-secure-contexts/#is-origin-trustworthy">
+              <dfn>Is origin potentially trustworthy</dfn></a>
+            </li>
+          </ul>
+        </li>
+
+        <li>
+          [[!SERVICE-WORKERS-1]] defines the following terms:
+          <ul>
+            <li>
+              <a href="https://w3c.github.io/ServiceWorker/#start-register-algorithm">
+              <dfn>Start Register</dfn></a>
+            </li>
+            <li>
+              <a href="https://w3c.github.io/ServiceWorker/#dfn-scope-url"><dfn>scope
+              URL</dfn></a>
+            </li>
+            <li>
+              <a href="https://w3c.github.io/ServiceWorker/#dfn-update-via-cache">
+              <dfn>ServiceWorkerUpdateViaCache</dfn></a>
+            </li>
+          </ul>
+        </li>
+
+        <li>
+          [[!ECMAS-402]] defines the following terms:
+          <ul>
+            <li>
+              <a href="https://www.ecma-international.org/ecma-402/1.0/#sec-6.2.2">
```

Comment:
same here? and others

Reply:
I opened one of these and it was a PDF

Reply:
One of the 402. This seems to link to a HTML site, so I can probably fix that one

Reply:
heh... let's see if that's been updated...

Reply:
Just checked, using "ecma-402" should "just work"™️